gui/default/index.html

```diff
@@ -325,7 +325,7 @@ <h4 class="panel-title">
                   <span ng-switch-when="idle"><span class="hidden-xs" translate>Up to Date</span><span class="visible-xs" aria-label="{{'Up to Date' | translate}}"><i class="fas fa-fw fa-check"></i></span></span>
                   <span ng-switch-when="syncing">
                     <span class="hidden-xs" translate>Syncing</span>
-                    <span ng-show="syncRemaining(folder.id)">({{syncPercentage(folder.id) | percent}}, {{syncRemaining(folder.id) | binary}}B)</span>
```

Comment:
This is some sort of behavior change, we now always show the span, presumably with "100%, 0 b" in the in sync case?

Reply:
Or potentially like `undefined %, NaN b` in some other cases :D

Reply:
> And I noticed the syncing status without a percentage at the end of a sync. In my opinion it's better to say that we are syncing at 100% (deleting stuff) than just syncing with no info, as the latter looks like there's still lots of work to do.

And is there any chance no folder summary was sent when pulling? That's the only reason needbytes may be undefined

Reply:
Deleting stuff should be at 95%, I think… I guess the status you’re seeing might be that we’ve got the latest folder summary but not yet the state change?

I guess things could  be undefined when starting up? The existing function checked for it, at least.

Reply:
That's one option, the other is that we pulled everything, so nothing is needed anymore, but the puller is still processing deletions, that it does at the end, and thus the state is still syncing

The 95% thing concerns remote devices only.

Sure they could, but when starting folder status is unknown, then scanning and only then possibly pulling. I can't imagine no summary to have been sent at that point. And index.html is full of unchecked references to model

Reply:
My feeling was that since the undefined condition was handled previously, perhaps it was for a reason. But perhaps that reason is obsolete or a misunderstanding.

Personally I don’t think "Syncing, 100%, 0 b" while we are still working is in any way more informative than just "Syncing", but it’s a minor point 🤷‍♂